### PR TITLE
Change ask timeout from Timeout to Duration in typed javadsl.

### PR DESCRIPTION
**Pull request by He-Pin**
refs:https://github.com/akka/akka/issues/25816

### CODE REVIEW

File: akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala [hunk 1]
```diff
@@ -167,7 +167,7 @@ trait ActorContext[T] extends akka.actor.typed.ActorContext[T] { this: akka.acto
    * *Warning*: This method is not thread-safe and must not be accessed from threads other
    * than the ordinary actor message processing thread, such as [[scala.concurrent.Future]] callbacks.
    */
-  def setReceiveTimeout(d: FiniteDuration, msg: T): Unit
+  def setReceiveTimeout(receiveTimeout: FiniteDuration, msg: T): Unit
```
Comment (patriknw) [inline akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala]: perhaps `timeout: ` to not repeat too much from the method name

**Review by patriknw — COMMENTED** [review]
good, one small nitpick

File: akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala (hunk 1 above)
Comment (hepin1989) [reply akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala]: I aliased it with the latter `responseTimeout` in the `ask` method.

Comment (hepin1989) [reply akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala]: @patriknw Done.

**Review by patriknw — APPROVED** [review]
LGTM

**Review by chbatey — APPROVED** [review]
LGTM, thanks!

**Pull request merged**